src/lib.rs

```diff
@@ -39,6 +39,7 @@
 //! serenity = "0.11"
 //! ```
 //!
+//! [`Cache`]: crate::caceh::Cache
```

Comment:
```suggestion
//! [`Cache`]: crate::cache::Cache
```

Reply:
Lol, that typo has been sitting there for 2 months.